src/main/java/com/users/repository/UserRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.users.repository;
+
+import com.users.entities.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
```

Comment:
de ce pe service se adauga adnotare si pe repository nu?

Reply:
pe Service se adauga acea adnotare si pentru o mai buna organizare a codului si a stii ca este o clasa unde se face business logic ul, am vazut ca lucreaza mai bine si cu alte adnotari precum @Transcation si @Autowired si este necesar si in contextul Spring pentru ca este un bean gestionat de acesta. Am citit ca nu este nevoie neaparat sa folosim @Repository, dar din nou este indicat pentru organizarea mai buna a codului si pentru a intelege ca este o clasa care comunica cu baza de date, la fel se foloseste cu @Transactional si este utila pentru a traducere mai buna a exceptiilor. O sa pun si eu la clasa mea